types/savings.go

```diff
@@ -36,5 +36,6 @@ type SavingsOverview struct {
 
 // This Structure will have the savings overview for each step and also detailed metrics in the future such as dlc metrics and gradle metrics
 type SavingsResponse struct {
-	Overview []SavingsOverview `json:"overview"` // array of SavingsOverview since one step can have multiple savings features enabled
+	Overview   []SavingsOverview `json:"overview"` // array of SavingsOverview since one step can have multiple savings features enabled
+	DlcMetrics dlc.Metrics       `json:"dlc_metrics"`
```

Comment:
Why is this required?

Reply:
To return dlc metrics to the UI: https://app.harness.io/ng/account/vpCkHKsDSxK9_KYfjCTMKA/module/code/orgs/default/projects/CIPRChecks/repos/harness-ti/pulls/225

Reply:
Added a new dlc metadata struct which I will use in SavingsResponse instead